### PR TITLE
Retrieve ingress domain

### PR DESCRIPTION
**Pull request by avisiedo**
Retrieve ingress domain for compose the IPA_SERVER_HOSTNAME variable.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,17 +14,15 @@ PARENT_IMG ?= $(shell source ci/config/env; echo $${PARENT_IMG})
...
 # Customizable cluster domain for deploying wherever we want
-# CLUSTER_DOMAIN ?= $(shell kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain')
-CLUSTER_DOMAIN_CMD = kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain'
+CLUSTER_DOMAIN_CMD = kubectl get dnses.config.openshift.io/cluster -o jsonpath='{.spec.baseDomain}'
 CLUSTER_DOMAIN ?= $(shell $(CLUSTER_DOMAIN_CMD))
-# REALM ?= APPS.$(shell echo $(CLUSTER_DOMAIN) | tr  '[:lower:]' '[:upper:]')
-REALM_CMD = echo "APPS.$(shell echo $(shell $(CLUSTER_DOMAIN_CMD)) | tr  '[:lower:]' '[:upper:]')"
+INGRESS_DOMAIN_CMD = kubectl get ingresses.config/cluster -o jsonpath='{.spec.domain}'
+INGRESS_DOMAIN ?= $(shell $(INGRESS_DOMAIN_CMD))
+REALM_CMD = echo "$(shell echo $(shell $(INGRESS_DOMAIN_CMD)) | tr  '[:lower:]' '[:upper:]')"
```
Comment (avisiedo) [inline Makefile:21]:
```suggestion
REALM_CMD = $(INGRESS_DOMAIN_CMD) | tr  '[:lower:]' '[:upper:]'"
```

File: Makefile [hunk 2]
```diff
@@ -14,17 +14,15 @@ PARENT_IMG ?= $(shell source ci/config/env; echo $${PARENT_IMG})
 # proof of concepts.
 
 # Customizable cluster domain for deploying wherever we want
-# CLUSTER_DOMAIN ?= $(shell kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain')
-CLUSTER_DOMAIN_CMD = kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain'
+CLUSTER_DOMAIN_CMD = kubectl get dnses.config.openshift.io/cluster -o jsonpath='{.spec.baseDomain}'
```
Comment (frasertweedale) [inline Makefile:17]: as discussed during review, if `CLUSTER_DOMAIN` is not used for anything, remove all references, checks, etc.

Comment (avisiedo) [reply Makefile]: Updated and checked.

File: Makefile [hunk 3]
```diff
@@ -14,17 +14,13 @@ PARENT_IMG ?= $(shell source ci/config/env; echo $${PARENT_IMG})
 # proof of concepts.
 
 # Customizable cluster domain for deploying wherever we want
-# CLUSTER_DOMAIN ?= $(shell kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain')
-CLUSTER_DOMAIN_CMD = kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain'
-CLUSTER_DOMAIN ?= $(shell $(CLUSTER_DOMAIN_CMD))
-# REALM ?= APPS.$(shell echo $(CLUSTER_DOMAIN) | tr  '[:lower:]' '[:upper:]')
-REALM_CMD = echo "APPS.$(shell echo $(shell $(CLUSTER_DOMAIN_CMD)) | tr  '[:lower:]' '[:upper:]')"
+INGRESS_DOMAIN_CMD = kubectl get ingresses.config/cluster -o jsonpath='{.spec.domain}'
```
Comment (tiran) [inline Makefile:17]: Command looks correct to me:

>  To display your default ingress domain, run the following command:
> ``oc get ingresses.config/cluster -o jsonpath={.spec.domain}``

https://docs.openshift.com/container-platform/4.10/networking/routes/route-configuration.html

**Review by tiran — APPROVED** [review]

**Pull request merged**